library/groovy

```diff
@@ -1,63 +1,32 @@
 Maintainers: Keegan Witt <keeganwitt@gmail.com> (@keeganwitt)
 GitRepo: https://github.com/groovy/docker-groovy.git
 
-
-# Groovy 3
-
-Tags: 3.0.14-jdk8, 3.0-jdk8, jdk8, 3.0.14-jdk8-jammy, 3.0-jdk8-jammy, jdk8-jammy
-Architectures: amd64, arm32v7, arm64v8, ppc64le
-GitCommit: bc4c3ee2a4e26f14458b121f238bb862c9de6b27
-Directory: jdk8
-
-Tags: 3.0.14-jdk11, 3.0-jdk11, jdk11, 3.0.14-jdk11-jammy, 3.0-jdk11-jammy, jdk11-jammy
-Architectures: amd64, arm32v7, arm64v8, ppc64le, s390x
-GitCommit: bc4c3ee2a4e26f14458b121f238bb862c9de6b27
-Directory: jdk11
-
-Tags: 3.0.14-jdk11-alpine, 3.0-jdk11-alpine, jdk11-alpine
-Architectures: amd64
-GitCommit: bc4c3ee2a4e26f14458b121f238bb862c9de6b27
-Directory: jdk11-alpine
-
-Tags: 3.0.14-jdk17, 3.0-jdk17, jdk17, 3.0.14-jdk, 3.0-jdk, 3.0.14, 3.0, 3, jdk, latest, 3.0.14-jdk17-jammy, 3.0-jdk17-jammy, jdk17-jammy, 3.0.14-jdk-jammy, 3.0-jdk-jammy, 3.0.14-jammy, 3.0-jammy, 3-jammy, jdk-jammy, jammy
-Architectures: amd64, arm32v7, arm64v8, ppc64le, s390x
-GitCommit: bc4c3ee2a4e26f14458b121f238bb862c9de6b27
-Directory: jdk17
-
-Tags: 3.0.14-jdk17-alpine, 3.0-jdk17-alpine, jdk17-alpine, 3.0.14-jdk-alpine, 3.0-jdk-alpine, 3.0.14-alpine, 3.0-alpine, 3-alpine, jdk-alpine, alpine
-Architectures: amd64
-GitCommit: bc4c3ee2a4e26f14458b121f238bb862c9de6b27
-Directory: jdk17-alpine
-
-
-# Groovy 4
-
-Tags: 4.0.7-jdk8, 4.0-jdk8, 4.0.7-jdk8-jammy, 4.0-jdk8-jammy
-GitFetch: refs/heads/4.0
+Tags: 4.0.8-jdk8, 4.0-jdk8, 4.0.8-jdk8-jammy, 4.0-jdk8-jammy
+GitFetch: refs/heads/master
 Architectures: amd64, arm32v7, arm64v8, ppc64le
-GitCommit: 085e0eb9a0611dd84c16efdcad26f352157d6f79
+GitCommit: de3d7a503dc033e4ce07942165aa363867ba9ce3
 Directory: jdk8
 
-Tags: 4.0.7-jdk11, 4.0-jdk11, 4.0.7-jdk11-jammy, 4.0-jdk11-jammy
-GitFetch: refs/heads/4.0
+Tags: 4.0.8-jdk11, 4.0-jdk11, 4.0.8-jdk11-jammy, 4.0-jdk11-jammy
+GitFetch: refs/heads/master
 Architectures: amd64, arm32v7, arm64v8, ppc64le, s390x
-GitCommit: 085e0eb9a0611dd84c16efdcad26f352157d6f79
+GitCommit: de3d7a503dc033e4ce07942165aa363867ba9ce3
 Directory: jdk11
 
-Tags: 4.0.7-jdk11-alpine, 4.0-jdk11-alpine
-GitFetch: refs/heads/4.0
+Tags: 4.0.8-jdk11-alpine, 4.0-jdk11-alpine
+GitFetch: refs/heads/master
 Architectures: amd64
-GitCommit: 085e0eb9a0611dd84c16efdcad26f352157d6f79
+GitCommit: de3d7a503dc033e4ce07942165aa363867ba9ce3
 Directory: jdk11-alpine
 
-Tags: 4.0.7-jdk17, 4.0-jdk17, 4.0.7-jdk, 4.0.7, 4.0, 4, 4.0.7-jdk17-jammy, 4.0-jdk17-jammy, 4.0.7-jdk-jammy, 4.0.7-jammy, 4.0-jammy, 4-jammy
-GitFetch: refs/heads/4.0
+Tags: 4.0.8-jdk17, 4.0-jdk17, 4.0.8-jdk, 4.0.8, 4.0, 4, 4.0.8-jdk17-jammy, 4.0-jdk17-jammy, 4.0.8-jdk-jammy, 4.0.8-jammy, 4.0-jammy, 4-jammy
```

Comment:
Did you want to move the `latest` and other generic tags that were on `3.0` images over to `4.0` images?

```diff
-groovy:alpine
-groovy:jammy
-groovy:jdk
-groovy:jdk8
-groovy:jdk8-jammy
-groovy:jdk11
-groovy:jdk11-alpine
-groovy:jdk11-jammy
-groovy:jdk17
-groovy:jdk17-alpine
-groovy:jdk17-jammy
-groovy:jdk-alpine
-groovy:jdk-jammy
-groovy:latest
```

Reply:
Thank you. Yes, I missed this. Added back.